mobile-app/app/screens/AppNavigator/screens/Portfolio/components/ActionButtons.tsx

```diff
@@ -110,13 +120,30 @@ function ActionButton(props: ActionButtonsProps): JSX.Element {
         onPress={props.onPress}
         testID={props.testID}
       >
-        <ThemedIcon
-          dark={tailwind("text-mono-dark-v2-900")}
-          light={tailwind("text-mono-light-v2-900")}
-          iconType={props.iconType}
-          name={props.icon}
-          size={props.iconSize ?? 24}
-        />
+        {props.iconType === undefined ? (
+          isLight ? (
+            <DFIIcon
+              width={props.iconSize}
+              height={props.iconSize}
+              color="#000"
+            />
+          ) : (
+            <DFIIcon
+              width={props.iconSize}
+              height={props.iconSize}
+              color="#FFF"
```

Comment:
We can simplify this to put the condition on the color right? Like <DFIIcon color={isLight ? "#000" : "#FFF"}